### DISCUSSION TITLE
is there checkout progressbar?

### DISCUSSION THREAD

**Discussion by Protoup20** [twbs/bootstrap#39541] [Q&A] [answered]
i don't know if i imaged it, but i could've swear i saw a checkout progress bar (its a line with color balls, the blue side extents to the next ball when you reach the next part of a form) on the bootstrap site. but i can't find it anymore.

progress tab does not contain it and i don't know where else it could be

**Comment by julien-deramond** [comment, marked as answer]
Hey @Protoup20 
Maybe the one you saw was this one https://getbootstrap.com/docs/5.3/utilities/position/#examples?

**Reply by Protoup20** [reply]
yes that's it, in knew i wasn't imagining things
